.drone/deploy/server/image/run.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+
+set -o errexit -o nounset
+
+export DRONE_SERVER_PORT=":${PORT}"
+
+/bin/drone-server "${@}"
```

Comment:
```suggestion
exec /bin/drone-server "${@}"
```
There's no need for the `sh` process to stick around.